source/manual/alerts/athena-fastly-logs-check.html.md

```diff
@@ -0,0 +1,109 @@
+GOV.UK CDN log data][govuk-athena]. It monitors this service by periodically
+querying each of the monitored databases and checking that they have > 0 results
+over the preceding hours.
+
+## Critical Alert
+
+A critical alert indicates that the database either has no recent results or
+that the process to query it failed. By looking at the Jenkins console output
+you should be able to determine which of these situations occurred.
+
+If the database query returned 0 results then this indicates that there
+is a problem with data being available to query. To determine the cause of
+this, you should check that:
+
+- [the Fastly CDN service is still sending logs to the expected S3 bucket and
+  these are in the expected JSON
+  structure](#how-to-check-log-file-presence-and-validity);
+- [the "GOV.UK fastly logs" crawler defined in AWS Glue doesn't have
+  errors in it's most recent logs](#how-to-check-the-govuk-fastly-logs-crawler);
+- whether you can [query the database yourself][query-athena] manually via
+  Athena to confirm there is no recent data.
+
+There is also the possibility that there are no results because there hasn't
+been any recent traffic to the particular environment. This is quite unlikely
+in production, but may be experienced in integration or staging if the GOR
+traffic replay isn't running.
+
+If the query failed with an error rather than returning 0 results then it is
+likely that we have data available but the monitoring is broken. In this case
+you should inspect the failed Jenkins job for clues as to what is not working.
+Potential causes could be: broken credentials, change in the AWS CLI tool
+syntax, or a change in the Athena API.
+
+### How to check log file presence and validity
+
+1. [Sign into the AWS console][aws-login] for the appropriate environment.
+1. Browse to [S3][] and find the `govuk-{environment}-fastly-logs` bucket.
+1. Navigate through the directories of the database in question to verify a
+   directory exists for the current day, for example
+   `govuk_www/year=2020/month=02/date=20`.
+1. If a directory exists for the current day check that there are files present
+   from the last few hours (the prefix search can make finding recent files
+   easier), new files should added approximately every 15 minutes and may
+   lag by 30 mins.
+1. If a recent log file is found then you can verify the JSON is valid using
+   [jq](https://stedolan.github.io/jq/).
+   1. Download the file through S3 web interface.
+   1. Run the file through `jq` to check for errors and to view the JSON
+      structure, for example
+      `gunzip -c 2020-02-20T19_45_00.000-nINz8eDOPAd3Uv1Mq-jW.log.gz | jq`.
+   1. If there are errors parsing the JSON you should check the contents of
+      the file to identify the problem.
+
+If you find that log files are not being written to S3 or the JSON is invalid
+you should log into the [Fastly control panel](https://manage.fastly.com)
```

Comment:
I don't know whether this is a blip, but `https://manage.fastly.com` doesn't load. Not sure if that's because this address has changed, or Fastly are having problems.

Reply:
Ah yes that's a problem with office DNS if I remember right. That somehow has been going on for 2 years 🤦‍♀